lambda/index.py

```diff
@@ -49,7 +49,7 @@ def _get_paginated_collection(function, key, page_token=None, collection=None):
     collection += response[key]
 
     if 'nextPageToken' in response:
-        get_paginated_collection(function, key, collection=collection,
+        __get_paginated_collection(function, key, collection=collection,
                                  page_token=response['nextPageToken'])
```

Comment:
Can you also run PEP8 against your change and make sure this indentation is still correct?